Library/Styles/BaseStyles.swift

```diff
@@ -14,13 +14,17 @@ public enum Styles {
   }
 }
 
+public func baseControllerFeedStyle <VC: UIViewControllerProtocol> () -> ((VC) -> VC) {
+  return VC.lens.view.backgroundColor .~ .ksr_grey_300
+}
```

Comment:
is there a need to have a separate concept of `baseControllerFeedStyle` and `baseTableControllerStyle`? should we ever use `baseTableControllerStyle` now?